meta-mender-core/recipes-mender/mender-client/mender-client_3.5.0.bb

```diff
@@ -1,4 +1,4 @@
-require mender-client.inc
+require mender-client-go.inc
```

Comment:
Q: What about older versions?

Reply:
For some reason they all have it already. I don't know why this was the exception.